posts/napari-cosmx-basics/using-napari-cosmx.qmd

```diff
@@ -0,0 +1,524 @@
+
+knitr::include_graphics("./figures/fig-duality.png")
+```
+
+<!-- napari-cosmx is flexible enough to quickly explore SMI data in a GUI -->
+<!--   yet robust enough to script reproducible results and tap into -->
+<!--   the underlying python objects.  -->
+
+# Introduction 
+
+This post is part of our napari series. In the [first blog post](../using-napari-for-cosmx-data/index.qmd){target="_blank"} I introduced the
+`napari-cosmx` plugin, how CosMx&#8482; Spatial Molecular Imager (SMI) data
+can be viewed as layers within napari, and a method for processing, 
+or "stitching", raw data that are exported from AtoMx&#8482; Spatial Informatics
+Portal (SIP). 
+
+One of the things that I love about the `napari-cosmx` plugin 
+is its duality. It's flexible enough to quickly explore SMI data in a Graphical
+User Interface (GUI) yet robust enough to script reproducible results and tap into
+the underlying python objects. In this post, I'll walk through some of the basic 
+ways in which we can use `napari-cosmx`to view SMI data. I'll make use of this
+duality and share a combination of GUI and programmatic tips-n-tricks.
+
+- @sec-example-data describes the example data that is publicaly available
+- @sec-working-with-layers shows you how to interact with viewer
+- @sec-scripting shows how we script to things to fine-tune image appearances and more
+
+::: {.callout-note}
+This is not intended to be official documentation for the `napari-cosmx` plugin. 
+The tips and tricks contained within are not an exhaustive list of features and methods.
+:::
+
+
+# The Example Dataset {#sec-example-data}
+
+The example dataset that I am using is the mouse coronal hemisphere
+FFPE that is available to download from NanoString's website [here](https://nanostring.com/products/cosmx-spatial-molecular-imager/ffpe-dataset/cosmx-smi-mouse-brain-ffpe-dataset/){target="blank"}. 
+
+::: {.callout-note}
+Large memory (RAM) might be required to work with raw images. Stitching the example data on a laptop might not work for everyone. 
+The raw data size for this example is 183 GBs. Not all raw data are needed to stitch
+and users can exclude the `CellStatsDir/Morphology3D` folder if downloading locally.
+If excluding this folder, the raw data is closer to 35 GBs. 
+
+The computer I used to stitch was an M1 Macbook Pro. Processing this 130 FOV, mouse
+1K data set took about 10 minutes, ~700%
+CPU, and a peak memory usage of about 12 GBs (swap space was also used). The size
+of the napari files combined was an additional 22 GBs of disk space. 
+:::
+
+Once downloaded, unzip the `HalfBrain.zip` file on your computer or external
+hard drive. Adjust the file structure to match the standard AtoMx 1.3.2 format. 
```

Comment:
It would be good to add a citation on Figure 13 of your 1st post, so people know what a standard format should look like.

Reply:
Added a cross reference but am not sure if it places or flows well. 

1839b5b31dc2642fc5facb6200534ea8760c9456

Reply:
Your latest commits have listed the data structure in the post and that's sufficient.